pkg/sippyserver/pr_commenting_processor.go

```diff
@@ -518,18 +518,30 @@ func buildNewTestRisksComment(sb *strings.Builder, jobRisks []*JobNewTestRisks,
 		SortByJobNameNT(notableJobRisks)
 		sb.WriteString(fmt.Sprintf("New Test Risks for sha: %s\n\n", sha))
 		sb.WriteString("| Job Name | New Test Risk |\n|:---|:---|\n")
+		rows := 0
 		for _, jr := range notableJobRisks {
 			for _, risk := range sortedTestRisks(jr.NewTestRisks) {
+				rows++
+				if rows > 20 {
```

Comment:
You could define a const for `20`, and then subtract `1` from it for the check on line 541.

Reply:
introduced `commentRowLimit`